### PR TITLE
Win 1421 zksync statics

### PR DESCRIPTION
**Pull request by ajays97**
This [commit](https://github.com/BitGo/BitGoJS/commit/897d969643174c80dc43fbdf6d63750c80644513) adds explorer constants which are needed for recovery.

This [commit](https://github.com/BitGo/BitGoJS/commit/145ea98ecda9e4b556ce0191ac2959f15a3a29e2) adds test cases for zketh module.

Ticket: WIN-1422

### CODE REVIEW

File: modules/sdk-coin-zketh/src/lib/resources.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ export const testnetCommon = EthereumCommon.custom(
   },
   {
     baseChain: 'sepolia',
+    hardfork: 'london',
```
Comment (gianchandania) [inline modules/sdk-coin-zketh/src/lib/resources.ts:12]: Also need to add eips: [1559]

File: modules/sdk-coin-zketh/src/lib/resources.ts [hunk 2]
```diff
@@ -20,5 +21,6 @@ export const mainnetCommon = EthereumCommon.custom(
   },
   {
     baseChain: 'mainnet',
+    hardfork: 'london',
```
Comment (gianchandania) [inline modules/sdk-coin-zketh/src/lib/resources.ts:24]: need to add eips: [1559]

File: modules/sdk-coin-zketh/src/lib/resources.ts (hunk 1 above)
Comment (ajays97) [reply modules/sdk-coin-zketh/src/lib/resources.ts:12]: Added

File: modules/sdk-coin-zketh/src/lib/resources.ts (hunk 2 above)
Comment (ajays97) [reply modules/sdk-coin-zketh/src/lib/resources.ts:25]: Added

**Review by margueriteblair — APPROVED** [review]

**Review by DinshawKothari — APPROVED** [review]

**Review by gianchandania — APPROVED** [review]

**Pull request merged**